worker-testing-integration/src/main/java/com/hpe/caf/worker/testing/validation/ValuePropertyValidator.java

```diff
@@ -16,9 +16,11 @@
 package com.hpe.caf.worker.testing.validation;
 
 import org.glassfish.jersey.internal.util.Base64;
+import org.joda.time.DateTime;
```

Comment:
We shouldn't be using JodaTime.  The fact that you didn't have to add a dependency to the pom for this is worrying.